src/modules/launcher/Wox.Plugin/AllowedLanguage.cs

```diff
@@ -24,8 +24,10 @@ public static bool IsAllowed(string language)
             ArgumentNullException.ThrowIfNull(language);
 
             // Using InvariantCulture since this is a command line arg
-            return language.ToUpper(CultureInfo.InvariantCulture) == CSharp.ToUpper(CultureInfo.InvariantCulture)
-                || language.ToUpper(CultureInfo.InvariantCulture) == Executable.ToUpper(CultureInfo.InvariantCulture);
+            var upperLanguage = language.ToUpper(CultureInfo.InvariantCulture);
```

Comment:
why ToUpper here? Let's use IgnoreCase as everywhere else

Reply:
At first I used StringComparison.InvariantCultureIgnoreCase. This gave CA1309: Use ordinal string comparison build error. I used InvariantCulture with this way. So I didn't change the existing code behavior